systemtest/src/test/java/io/strimzi/systemtest/topic/TopicST.java

```diff
@@ -93,6 +93,8 @@ void testMoreReplicasThanAvailableBrokers() {
 
         assertThat("Topic exists in Kafka itself", hasTopicInKafka(newTopicName));
         assertThat("Topic exists in Kafka CR (Kubernetes)", hasTopicInCRK8s(kafkaTopic, newTopicName));
+
+        KafkaTopicResource.deleteKafkaTopicWithoutWait(kafkaTopic);
```

Comment:
Why do we need to delete topic explicitly here?

Reply:
Well my idea here was the topic can affect the test, but it's nonsense, I'm gonna delete it